VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/consultation-modals/consultation-leave/consultation-leave.component.scss

```diff
@@ -1,4 +1,23 @@
+@import 'abstracts';
+
 .modal-content {
   text-align: left;
   padding: 50px;
 }
+.consultation-modal {
+  position: fixed;
+  left: 50%;
+  top: 70%;
+  width: 35vw;
```

Comment:
This width will never be used as it will be overridden by the subsequent ones in the media query.